server/command.go

```diff
@@ -511,11 +520,15 @@ func (p *Plugin) subscriptionsAddCommand(ctx context.Context, info *gitlab.UserI
 		ctx, info, fullPath, config.EnablePrivateRepo)
 
 	if err != nil {
-		if errors.Is(err, gitlab.ErrNotFound) {
+		switch {
+		case errors.Is(err, gitlab.ErrNotFound):
 			return "Resource with such path is not found."
-		} else if errors.Is(err, gitlab.ErrPrivateResource) {
+		case errors.Is(err, gitlab.ErrPrivateResource):
 			return "Requested resource is private."
+		case strings.Contains(err.Error(), invalidTokenError):
+			p.handleRevokedToken(info)
```

Comment:
The code cleanup is good here, though I think for the purpose of this PR, can we just add a call to `handleGitlabError(err)` at the top of the `if err != nil` block?

Reply:
Yeah I'll do that.